configs/components/leatherman.rb

```diff
@@ -94,7 +94,7 @@
 
   # Make test will explode horribly in a cross-compile situation
   # Tests will be skipped on AIX until they are expected to pass
-  if platform.architecture == 'sparc' || platform.is_aix? || platform.is_huaweios?
+  if platform.architecture == 'sparc' || platform.architecture == 's390x' || platform.is_aix? || platform.is_huaweios?
```

Comment:
No. Once this gets merged I intend to start working on that. Then the refactoring can begin.